Netbox/legos/netbox_get_handle/README.md

```diff
@@ -0,0 +1,25 @@
+[<img align="left" src="https://unskript.com/assets/favicon.png" width="100" height="100" style="padding-right: 5px">](https://unskript.com/assets/favicon.png) 
+<h2>Get Netbox Handle</h2>
+
+<br>
+
+## Description
+This Lego Get Nomad Handle.

```

Comment:
it says nomad, but this is for netbox, right?

Reply:
Changed